### ISSUE TITLE
Circe Encoders and Decoders Duplication

### ISSUE THREAD

**Issue by calvinbrown085** [guardrail-dev/guardrail#92]
```implicit def decodeInstant: Decoder[Instant] = j8time.decodeInstant.or(decodeLong.map(Instant.ofEpochMilli))```

We don't actually need to generate these, these come free if we extend `TimeInstances`

https://github.com/circe/circe/blob/master/modules/java8/src/main/scala/io/circe/java8/time/TimeInstances.scala#L28

Also I can see this being a potential problem for some people since this requires an extra dep from circe.
Just a heads up!

**Comment by blast-hardcheese** [comment]
Good call! Additionally, I'm realizing the epochMillis conversion should be enabled explicitly instead of assuming the user wants that behavior

**Comment by blast-hardcheese** [comment]
Related to #103
